extensions/cornerstone-dicom-seg/src/utils/_hydrateSEG.ts

```diff
@@ -62,6 +60,9 @@ async function _hydrateSEGDisplaySet({
     }
   });
 
+  // Do the entire update at once
```

Comment:
As it says, this change just does the entire update in a single operation, preventing inconsistencies and odd rendering effects.